multi-agent/lib/mas/elements/nodes/common/capabilities/hitl_capable.py

```
@@ -0,0 +1,91 @@
+      - ``_approval_policy``    (injected at runtime by NodeRuntimeBinder)
+
+    The binder injects gate/policy via the ``SupportsHITL`` protocol
+    which checks for ``set_approval_gate`` and ``set_approval_policy``.
+    """
+
+    def __init__(
+        self,
+        *,
+        hitl_mode: HITLMode = HITLMode.SKIP,
+        execution_holder: Optional[ExecutionContextHolder] = None,
+        **kwargs: Any,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)